orcid-core/src/main/java/org/orcid/core/manager/impl/SalesforceManagerImpl.java

```diff
@@ -0,0 +1,124 @@
+package org.orcid.core.manager.impl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.Resource;
+
+import org.codehaus.jettison.json.JSONArray;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+import org.orcid.core.manager.SalesforceManager;
+import org.orcid.core.salesforce.adapter.SalesForceAdapter;
+import org.orcid.core.salesforce.dao.SalesforceMicroserviceClient;
+import org.orcid.core.salesforce.model.Integration;
+import org.orcid.core.salesforce.model.Member;
+import org.orcid.core.salesforce.model.MemberDetails;
+import org.orcid.core.salesforce.model.Opportunity;
+import org.orcid.core.salesforce.model.SlugUtils;
+import org.orcid.core.salesforce.model.SubMember;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SalesforceManagerImpl implements SalesforceManager {
+
+    @Resource
+    private SalesForceAdapter salesForceAdapter;
+
+    @Resource
+    private SalesforceMicroserviceClient client;
+
+    public List<Member> retrieveMembers() {
+        List<Member> members = new ArrayList<Member>();
+
+        try {
+            String membersListString = client.retrieveMembers();
+            JSONObject membersListJsonObject = new JSONObject(membersListString);
+            members = salesForceAdapter.createMembersListFromJson(membersListJsonObject);
+        } catch (IOException | InterruptedException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Comment:
I think it makes sense to use log4j logger to log errors as we do through our source code ( lines 41, 45, 116 and 119)